userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -99,26 +119,41 @@ Kill Bill Administrator (or Kill Bill Admin):: The person responsible for instal
 
 Payment Method:: A record of payment details, such as for credit cards, debit cards, PayPal accounts, etc. Each account can have many payment methods and one default payment method. Note that much of the representation of a payment method is plugin-specific and is actually stored by the payment plugin itself.
 
+PCI Compliance
+{ PC​I DSS (payment card industry data security standard)}
+{The payment card industry data security standard (PCI DSS) is a set of requirements designed to ensure that all companies that process, store, or transmit credit card information maintain a secure environment. Zuora has compliance in accordance with PCI Service Provider Level1 (PCI DSS SP L1) v1.2. Zuora is listed on the VISA website as an approved provider.}
```

Comment:
We should probably not document Zuora compliance in our docs 🙃

Reply:
Oh... I see what happened. I copied/pasted that info from somewhere, intended to rephrase/edit it, and I forgot.

Reply:
@pierre or @reshmabidikar - I have commented out the above "PCI Compliance" for now so that Zuora doesn't show up. Please review.